community/kernel/src/main/java/org/neo4j/kernel/internal/KernelDiagnostics.java

```diff
@@ -122,10 +134,37 @@ public int compare( File o1, File o2 )
                 logger.log( fileInformation );
 
                 total += size;
+                if ( ++count == MAX_DIRECTORY_LISTING_THRESHOLD )
```

Comment:
i do not think we should do this, yes it will remove some logging but also can consume so important information for future investigation